IBPSA/BoundaryConditions/Types.mo

```diff
@@ -9,8 +9,7 @@ package Types "Package with type definitions"
 <p>
 Enumeration to define the data source used in the weather data reader.
 </p>
-</html>", revisions=
-"<html>
+</html>", revisions="<html>
```

Comment:
Try to avoid these model changes that do not actually contain any significant code changes. Especially if they make the formatting worse. You can use 'git checkout master --  IBPSA/BoundaryConditions/Types.mo' and commit to revert this change.